src/routes.rs

```diff
@@ -105,6 +106,15 @@ fn get_homepage<'a>() -> HTML<'a> {
     HTML(ROOT_RESPONSE)
 }
 
+/*
+* Trow health endpoint
+* GET /healthz
+ */
+#[get("/healthz")]
+fn get_healthz() -> JsonValue {
```

Comment:
I think I'd prefer just `healthz` as the function name.

I added get/put etc to the OCI endpoints as it makes it easier to go between the spec and the code. This does mean `get_homepage` and `get_v2root` are wrong as well, but we can worry about that later.

Reply:
Ok will update this